skxray/fitting/xrf_model.py

```diff
@@ -479,12 +497,13 @@ def set_ratio(self, item, option=None):
         data_list = [str(item)+'_'+str(v).lower() for v in data_list]
 
         for linename in data_list:
+            param_name = str(linename) + '_ratio_adjust'
             if linename not in self.element_line_name:
                 continue
             new_val = element_dict['ratio'].copy()
             if option:
-                new_val['adjust_element'] = option
-            self.new_parameter.update({str(linename)+'_ratio_adjust': new_val})
+                self._element_strategy[param_name] = option
+            self.new_parameter.update({param_name: new_val})
```

Comment:
So you basically create a new _element_strategy dict to store element related information only. This may have some advantages because element strategy are independent to other strategies, so we don't need to combine them together. 
Lots of my codes are based on the old default_param dict, so I will think about the mitigation process with this new structure.